syntax = "proto3";
package ops_automation;

option go_package = "go.easyops.local/contracts/protorepo-models/easyops/model/ops_automation";





/*
创建巡检菜单
*/
message Menu {
    /*
    菜单名称
    */
    string name = 1 ;
    /*
    菜单图标
    */
    string icon = 2 ;
    /*
    菜单分类
    */
    string category = 3 ;
    /*
    访问白名单 [easyops, test_uesr]
    */
    repeated string visitors = 4 ;
    /*
    管理白名单 [easyops, test_uesr]
    */
    repeated string managers = 5 ;
    /*
    菜单Id
    */
    string id = 6 ;
}

